tests/test_pull_requests.py

```diff
@@ -0,0 +1,124 @@
+from github_to_sqlite import utils
+import pytest
+import pathlib
+import sqlite_utils
+from sqlite_utils.db import ForeignKey
+import json
+
+
+@pytest.fixture
+def pull_requests():
+    return json.load(open(pathlib.Path(__file__).parent / "pull_requests.json"))
+
+
+@pytest.fixture
+def db(pull_requests):
+    db = sqlite_utils.Database(memory=True)
+    db["repos"].insert(
+        {"id": 1},
+        pk="id",
+        columns={"organization": int, "topics": str, "name": str, "description": str},
+    )
+    utils.save_pull_requests(db, pull_requests, {"id": 1})
+    return db
+
+
+def test_tables(db):
+    assert {"pull_requests", "users", "repos", "milestones"} == set(
+        db.table_names()
+    )
+    assert {
+        ForeignKey(
+            table="pull_requests", column="repo", other_table="repos", other_column="id"
+        ),
+        ForeignKey(
+            table="pull_requests",
+            column="milestone",
+            other_table="milestones",
+            other_column="id",
+        ),
+        ForeignKey(
+            table="pull_requests", column="assignee", other_table="users", other_column="id"
+        ),
+        ForeignKey(
+            table="pull_requests", column="user", other_table="users", other_column="id"
+        ),
+    } == set(db["pull_requests"].foreign_keys)
+
+
+def test_pull_requests(db):
+    pull_request_rows = list(db["pull_requests"].rows)
+    assert [
+        {
+            'id': 313384926,
```

Comment:
Minor detail: I use Black for this repo, which requires double quotes - running "black ." in the root directory (with the latest version of Black) should handle this for you.

Reply:
ran black as suggested